autocomplete_light/registry.py

```diff
@@ -173,12 +176,20 @@ def _register_model_autocomplete(self, model, autocomplete=None,
 
         if base.search_fields is None and 'search_fields' not in kwargs:
             try:
-                model._meta.get_field('name')
-            except:
-                raise Exception('Add search_fields kwargs to .register(%s)'
-                                % model.__name__)
-            else:
-                kwargs['search_fields'] = ['name']
+                search_fields = admin.site._registry[model].search_fields
```

Comment:
We might be using the wrong site though, we need the admin_site from there: https://github.com/django/django/blob/stable/1.8.x/django/contrib/admin/widgets.py#L241